extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -113,8 +117,10 @@ public function renderInfobox( $text, $params, $parser, $frame ) {
 			return $this->handleError( wfMessage( 'portable-infobox-unimplemented-infobox-tag', [ $e->getMessage() ] )->escaped() );
 		} catch ( \Wikia\PortableInfobox\Parser\XmlMarkupParseErrorException $e ) {
```

Comment:
Why we don't use `use` for this one but do for e.g. `InvalidInfoboxParamsException`? Let's be consistent.